fluent-syntax/test/fixtures_behavior/section_with_no_nl_after_it.ftl

```diff
@@ -1 +0,0 @@
-[[ This is a correct section ]]
```

Comment:
It would be good to still test that this parses alright, even if it parses to an empty AST at this point.